sycl/source/detail/scheduler/graph_builder.cpp

```diff
@@ -1001,7 +1018,6 @@ Scheduler::GraphBuilder::addCG(std::unique_ptr<detail::CG> CommandGroup,
   // FIXME employ a reference here to eliminate copying of a vector
   std::vector<DepDesc> Deps = NewCmd->MDeps;
   for (DepDesc &Dep : Deps) {
-    Dep.MDepCommand->addUser(NewCmd.get());
```

Comment:
I can't get why this should be removed.

Reply:
it is moved to the deps building cycle above (line 1011 on the right)